Source/Core/Core/PowerPC/JitArm64/JitAsm.cpp

```diff
@@ -244,6 +228,24 @@ void JitArm64::GenerateAsm()
   FlushIcache();
 }
 
+void JitArm64::EmitUpdateMembase()
+{
+  LDR(IndexType::Unsigned, MEM_REG, PPC_REG, PPCSTATE_OFF(mem_ptr));
+}
+
+void JitArm64::EmitStoreMembase(const ARM64Reg& msr)
```

Comment:
JitAsm.cpp is generally for code that only gets emitted once. These two functions would be a better fit for Jit.cpp.